src/lib-components/lists/Table.vue

```diff
@@ -96,8 +96,12 @@ const loadAndRender = (): void => {
     q: query.value,
   }
 
-  BaseAPI.get(props.tableData.url, { skipLoader: !props.loader }, params).then(
-    (success: AxiosResponse) => {
+  BaseAPI.get<TrailsResponsePaged<unknown>>(
```

Comment:
The reason `Trails` prefixed interfaces ultimately made it into Trees.  Because Trees loves Trails and Trails love Trees.